product_form_sale_link/views/product_template.xml

```diff
@@ -10,12 +10,12 @@
         <field name="groups_id" eval="[(4, ref('sales_team.group_sale_salesman'))]"/>
         <field name="inherit_id" ref="sale.product_template_form_view_sale_order_button"/>
         <field name="arch" type="xml">
-            <button name="action_view_sales" position="after">
+            <xpath expr="//form/sheet/div[@class='oe_button_box']" position="inside">
```

Comment:
Hi, 
replace : 

``<xpath expr="//form/sheet/div[@class='oe_button_box']" position="inside">``

by 

``<xpath expr="//form/sheet/div[hasclass='oe_button_box']" position="inside">``

to make runbot happy. (ref : https://runbot2-3.odoo-community.org/runbot/static/build/3409921-1072-5ffc6f/logs/job_20_test_all.txt)

Reply:
hi,
with hasclass in xpath travis get this error:
ValueError: Element '<xpath expr="//form/sheet/div[@hasclass='oe_button_box']">' cannot be located in parent view